.npmrc

```diff
@@ -1 +1,2 @@
-strict-peer-dependencies=false
+# pnpm - storybook requires hoisting
+public-hoist-pattern[]=*storybook*
```

Comment:
Let's remove this, as described in another comment, I want this project to demonstrate the problems, and this is a workaround.